extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -357,32 +350,6 @@ export class AmpScrollableCarousel extends BaseCarousel {
     }
   }
 
-  /**
-   * @param {number} newPos
-   * @param {number} oldPos
-   * @private
-   */
-  updateInViewport_(newPos, oldPos) {
-    const seen = [];
-    this.withinWindow_(newPos, (cell) => {
-      seen.push(cell);
-      Services.ownersForDoc(this.element).updateInViewport(
-        this.element,
-        cell,
-        true
-      );
-    });
-    if (oldPos != newPos) {
-      this.withinWindow_(oldPos, (cell) => {
-        if (!seen.includes(cell)) {
-          const owners = Services.ownersForDoc(this.element);
-          owners.updateInViewport(this.element, cell, false);
-          owners.schedulePause(this.element, cell);
```

Comment:
Oops. Did we track this anywhere? `schedulePause` specifically?

Reply:
what about `schedulePause`? For this PR, I didn't notice the schedulePause here 😬  and can revert that change so I don't remove intended functionality

Reply:
I think we have to, yes. We can remove `updateInViewport` of course. But we haven't gotten to pause/resume yet.